CTCommons/RabbitMQ/RabbitMQConnection.cs

```diff
@@ -32,12 +46,24 @@ public RabbitMQConnection(ILogger<RabbitMQConnection> logger)
             {
                 HostName = Globals.appSettings.RabbitMQServer,
                 UserName = Globals.appSettings.ADMIN_USER_ID,
-                Password = Globals.appSettings.ADMIN_PASSWORD
+                Password = Globals.appSettings.ADMIN_PASSWORD,
+                Port = 5672
```

Comment:
Might be a good idea to move to Globals.appSettings, probably not great for a port number to be hardcoded

Reply:
Agreed. While it's not a problem for containers (the mapping can occur at the boundary) it's still useful to be able to specify an arbitrary port number. AppSeettings now defaults to 5672 if it is unset.

Reply:
Port number added to appsettings in
https://github.com/classtranscribe/WebAPI/pull/101/commits/69d23ab623b1519f7809a48002b9f962ee8cfc8d